doc/MetadataAndQuery.md

```diff
@@ -0,0 +1,312 @@
"tz1NGYqqrHpE5GLb1dHieMwred7SzUHDidQz", "tz1LePMD9gxvbGMgQStoCvWAw4eCHfvqv5tH", "tz1WESA74Lm7UQfFS1Fm4qGimK6dtitU99i8", "tz1eGKPkRKc8dCa8gzs5NXtCNjpyQVsu6jap", "tz1g3Hb1GojyuTFCMvFW4i1mUG3cGkqvCYKY", "tz1fU53kw34jGCM66gVUhtxrEvnV7Ajac5bW", "tz1enAY9gv133dWioFe24jwcs1ps4mXFuFKA", "tz1PSoMfjLCVaDnBDNpDMGS7DfNRdeTFpyv2", "tz1fyYJwgV1ozj6RyjtU1hLTBeoqQvQmRjVv", "tz1LhS2WFCinpwUTdUb991ocL2D9Uk6FJGJK", "tz1cpE5LTaV3HkwbbcFiYxvjxyqGNtBaqe2s", "tz1UT7vYBX5eY8gYX587FNW3TrNBftRKjbLM", "tz1LTFTfpFXP6GyQmkh6Tnr11TrcpFhM2ZyA", "tz1bGcJxAvh75GAwRp9Gedur6z2hEci64a9x", "tz1UmPE44pqWrEgW8sTRs6ED1DgwF7k43ncQ", "tz1btT98KZWK99WbQfvvnBSsveJp1YY69iqm", "tz1hjzSgyKDDm5FHZTRRd7xpxYSVwuZNjXTq", "tz1a3sMhA1wvYMQz8HGkNqSCoQ44X1yoT7w6", "tz1bhL4zwmLJvHJK5ejDDKdeatpqorvJdc2s", "tz1ZNFHPbkBEY5CWjtQ8GEzfUw6x8SXVC3zQ", "tz1LUFr9jQZqKJCoYBnobkWAXHiFPy6DQcUj", "tz1LzHyPBxieoLfoQT4YK7EGFCehQEj56X9i", "tz1PrWh3v16P99EBTPV92WgzUWqEescYN53J", "tz3e5mfRyZBcTcuerCrbxjtXW5bDyvA8wmPU", "tz1Y3JGezuAgo1YC2Tv8H4835gp1s3dpQYt6", "tz1M1nnzd8HQtt3VDdK2Su9yDmJisCa9N1aC", "tz1WcbghSgAKqjtJDBJrsQrzxPb6zwcMjLVk", "tz1X1y4yJKAih7Wwgxaun4QTEyzFb7sVQTWr", "tz2LgN1uN7AUwtdnGC4Bu8LGzHiTw5QQhCdQ", "tz1RzkGFKxqggJGhs18M3ejAkwnYNa3bNbFt", "tz1awU3Uo2F45cXKaY9X7KkucLyas76dpeCf", "tz1ZnDhjUTZ8LURHiCPWLP4ayYYYyV5Ddqyz", "tz1amfhHn47i5ZYVnUGsTodsZW6G52vqAThE", "tz1a4rq7TnGYr4cjN1FPGLh3voLUiknBYVfA", "tz1SJ74H8C3g9YdhpMdRqvsE8QeaQYESdfEq", "tz1SN46ZFMycvnPJFM37xxk24wK4wDT2KbZe", "tz1XV5grkdVLMC9x5cy8GSPLEuSKQeDi39D5", "tz1iLVCjraxVZm584zJzyfMWAWZCwt9QjfR3", "tz1XmX2QYNYtkTxtMYL7iaGgKS8vxaLRCPpE", "tz1d4fF7w2kXGNxUwuPUqzSWQG25Zs2mxJyV", "tz1QoVcn2FFC9CoFe59we38F8KNvv9yia6mC", "tz1NVUTRpHUoy9t4DEiBKCeyUaBjs8v8N8NB", "tz1SjzoafiHQ8yYBAQLzmggsew2VFMnEAtgD", "tz1MEVmsoxhTTukrbv45dZEvJC11ptDyrsBx", "tz1PSTFZBezQRMMV2Zk295L9pCu1qEF6k1o7", "tz1ZwkHmBrQwdFze1ZD49kBakeJ1JBQ4Rrt2", "tz1b7X44b7YmpfuDCJfgyYij4Hc4B6e7J5da", "tz3gN8NTLNLJg5KRsUU47NHNVHbdhcFXjjaB", "tz1aWXP237BLwNHJcCD4b3DutCevhqq2T1Z9", "tz1ZaMdaE9Qp3eU6acsQ8d4vvub4As8KeJJk", "tz2PVx7pTeTNaxWLzq2JAYf6NaAGg2um2LPJ", "tz1iAjuR4MMnf4bZo9DHJrsCBuLxMb9jdgAX", "tz1f8vTXY3jm3jKXnArjYjQ9AG7PVinDFvdF", "tz1funkyE3bAnfmaarN41iaa5fJC3W5ProGk", "tz1YCABRTa6H8PLKx2EtDWeCGPaKxUhNgv47", "tz1QJZDkPuajnSXHHkCgynxjsQBe7wxkvwgY", "tz1fhYDrVBxV7tEnDMAtgTFqGZcy4GNAybnN", "tz1b2rPGAQwvX4whvVNVZYMTBbk3Bwhu6vM3", "tz1c5zSbuq9qfNwHq2C4gjcRPWgFTXx1ZGX4", "tz1SazLSmCwRohSgfU85oqCLr5mvSsmPCFgH", "tz1XNrqWhTydoUB3qyefVZRs4HbmJahkmzVZ", "tz1URf65gvzo5onDViCAAC2PKBwYduAdH2Z5", "tz3bkFdXym6mHqJPkHHqbQMbesU5EViHEKmb", "tz1PygQHREmzb5aFQ4wKFjdbvTgS8ay7di85", "tz1h59SmGuwbjVispYe56fBh2LGDRHK6HxmL", "tz1eUsgK6aj752Fbxwk5sAoEFvSDnPjZ4qvk", "tz1foaS8v7zv3d3ANufKqjjqaVLbrnk4VGoQ", "tz1WKKjZnVHe4CtthLVPjGzEAFGb8BEFL3Uz", "tz1bxVgPCiveFHSand7zn22S8ZBtvNQbqgqU", "tz1MTh1BLcJHmdeEaEMr8Qy3SJgocM8MEu1N", "tz1TJhs1Sb81YxvQfMcT27q8x4a7MCCRq3BS", "tz1V7Vmvq2PbcKyL3sJZ8LYKDaFHRGPWCGqu", "tz1SfH1vxAt2TTZV7mpsN79uGas5LHhV8epq", "tz1W4QbdQsYDWVifj7kTLVEqpJVoLaW1qFZW", "tz1ZashkRLkUtJvHCijGSkaLppaRfiNEivuk" ]
```

Comment:
Maybe this can be elided? Do we need all the values?

Reply:
I just wanted to show full, example but sure, I can add something like `(...)` in the middle